options.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 	errors        *Errors
 
 	reverseHandlers       []clientHandler
+	namespaceDelimiter    string
```

Comment:
Seems to miss a function to set this from outside the library (e.g. `WithNamespaceDelimiter`), same for the handler side

Reply:
that would be important, will add today! 😂